packages/grid_client/src/modules/contracts.ts

```diff
@@ -98,6 +98,12 @@ class Contracts {
     return this.client.contracts.getContractIdByActiveRentForNode(options);
   }
 
+  @expose
+  @validateInput
+  async contractLock(options: ContractConsumption) {
```

Comment:
please implement the model for it
```suggestion
  async contractLock(options: ContractLockModel) {
```

Reply:
updated